web/package.json

```diff
@@ -25,10 +25,16 @@
     "@angular/router": "^5.2.11",
     "@angular/service-worker": "^5.2.11",
     "angular2-notifications": "^1.0.2",
+    "bootstrap": "^4.3.1",
     "core-js": "^2.5.7",
     "http-status-codes": "^1.3.0",
     "intl": "^1.2.5",
+    "j": "^1.0.0",
+    "jquery": "^3.3.1",
+    "node-sass": "^4.11.0",
+    "popper.js": "^1.14.7",
     "rxjs": "^5.5.11",
+    "tether": "^1.4.5",
```

Comment:
We didn't have this dependency before as well. I strongly believe you should add only same stuff we used before. It's bad practice to add unused code to dependency. It's increase final file size and page loading time.